web/styles/popovers.css

```diff
@@ -409,7 +409,7 @@ ul {
         .group-name {
             font-weight: bold;
 
-            .zulip-icon {
+            .zulip-icon-triple-user {
```

Comment:
`triple-user` -> `triple-users`

Should this change be in previous commit?

Reply:
Thank you for pointing it out. The commits have been updated accordingly.

Reply:
Wasn't updated.

Reply:
Done.